workloads/virtual-machines/interfaces-and-networks.md

```diff
@@ -18,34 +18,87 @@ All possible configuration options are available in the
 [Interface API Reference](https://kubevirt.io/api-reference/master/definitions.html#_v1_interface)
 and [Network API Reference](https://kubevirt.io/api-reference/master/definitions.html#_v1_network).
 
-> **Note:** Currently, the only supported network type is `pod`.
-
 ## Backend
 
 Network backends are configured in `spec.networks`. A network must have a
 unique name. Additional fields declare which logical or physical device the
 network relates to.
 
+Each network should declare its type by defining on of the following fields:
+
+| Type | Description |
+|--|--|
+| `pod` | Default Kubernetes network |
+| `multus` | Secondary network provided using Multus |
+
+### pod
+
 A `pod` network represents the default pod `eth0` interface configured by
-cluster SDN solution that is present in each pod.
+cluster network solution that is present in each pod.
+
+```yaml
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: default
+          bridge: {}
+  networks:
+  - name: default
+    pod: {} # Stock pod network
+```
+
+### multus
+
+It is also possible to connect VMI to a secondary networks using
+[Multus](https://github.com/intel/multus-cni). This assumes that multus is
+installed accross your cluster and a corresponding
+`NetworkAttachmentDefinition` CRD was created.
 
-> **Note:** Currently, the only supported network type is `pod`. In the future,
-> more network types may be supported.
+First of all, a `NetworkAttachmentDefinition` for the desired secondary network
+must be created. Following example defines a network which uses the [ovs-cni
+plugin](https://github.com/kubevirt/ovs-cni), which will connect the VMI to Open
+vSwitch bridge `br1` and VLAN 100. Other CNI plugins such as ptp, bridge,
+macvlan or flannel might be used as well, for their installation and usage refer
+to respective project documentation.
+
+`NetworkAttachmentDefinition` is usually created by the administrator and just
```

Comment:
First the `NetworkAttachmentDefinition` needs to be created. That is usually done by an administrator. Users can then reference the definition.